game-backend/src/game-engine/intent/ConsumeIntent.ts

```diff
@@ -22,10 +22,15 @@ export default class ConsumeIntent implements Intent {
     return Math.min(health + FOOD_REGENERATION_VALUE, MAX_HEALTH);
   }
 
-  private static getConsumableInReach(state: State, unit: Unit): Food | null {
+  private static getConsumableInReach(
+    state: State,
+    unit: Unit
+  ): Consumable | null {
     const objectsOnPosition = getObjectsOnField(state, unit.position);
     const consumable = objectsOnPosition
-      ? objectsOnPosition.find(obj => obj.type === GAME_OBJECT_TYPES.FOOD)
+      ? (objectsOnPosition.find(
+          obj => obj.type === GameObjectTypes.Food
```

Comment:
Shouldn't this now be any consumable? (Totally fine to do it with the first powerup implementation - just remeber to keep an eye on this :v:)